e2e-tests/trade-finance-manager/cypress/integration/journeys/case/amendments/1. add-amendment-request.spec.js

```diff
@@ -103,6 +103,34 @@ context('Amendments - Request date', () => {
     amendmentsPage.errorMessage().contains('Amendment request date cannot be in the future');
   });
 
+  it('should return errors when clicking continue on year in wrong format', () => {
+    cy.login(PIM_USER_1);
```

Comment:
Can we please enter test case with year as  a string and year with spaces please.

Reply:
done